### PR TITLE
fix(prosemirror-model): fix textBetween

### PR DESCRIPTION
**Pull request by ocavue**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
 
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [<<url here>>](https://prosemirror.net/docs/ref/#model.Node.textBetween)
image

- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/prosemirror-model/index.d.ts [hunk 1]
```diff
@@ -1409,7 +1420,7 @@ export interface DOMOutputSpecArray {
     8?: DOMOutputSpec | 0 | undefined;
     9?: DOMOutputSpec | 0 | undefined;
 }
-export type DOMOutputSpec = string | Node | DOMOutputSpecArray | {dom: Node, contentDOM?: Node | undefined};
+export type DOMOutputSpec = string | Node | DOMOutputSpecArray | { dom: Node; contentDOM?: Node | undefined };
```
Comment (AnthonyWeston) [inline types/prosemirror-model/index.d.ts:1423]: Was this semicolon here inserted by accident? Other than that, looks good to me.

**Review by AnthonyWeston — CHANGES_REQUESTED** [review]

File: types/prosemirror-model/index.d.ts (hunk 1 above)
Comment (ocavue) [reply types/prosemirror-model/index.d.ts:1423]: This change is intentional. I used Prettier to format this file, according to [DefinitelyTyped's request](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes). TypeScript accepts both `,` and `;` here, but Prettier prefers `;`.

Comment (AnthonyWeston) [reply types/prosemirror-model/index.d.ts:1423]: Huh, I didn't know that. Nifty 👍

**Review by AnthonyWeston — APPROVED** [review]

**Comment by AnthonyWeston** [comment]
Ready to merge

**Comment by ocavue** [comment]
@AnthonyWeston Thank you for your review.

**Pull request merged**